### PR TITLE
swapped emojis with icons

### PR DESCRIPTION
**Pull request by 4rtemis-4rrow**
swapped emojis with icons, this provides a much cleaner look

this could still be improved by adding a commandline argument to choose either emojis or icons, but I don't know rust, and me trying to implement this will cause disasters

### CODE REVIEW

File: src/constants.rs [hunk 1]
```diff
@@ -1,64 +1,64 @@
...
+    (284, " "),
+    (293, "🌧"),
+    (296, "🌧"),
+    (299, "🌧"),
+    (302, "🌧"),
+    (305, "🌧"),
+    (308, "🌧"),
+    (311, "🌧"),
+    (314, "🌧"),
+    (317, "🌧"),
```
Comment (khaneliman) [inline src/constants.rs:28]: should update to reflect the change to 179

File: src/constants.rs [hunk 2]
```diff
@@ -1,64 +1,64 @@
...
+    (179, " "),
+    (182, " "),
+    (185, " "),
+    (200, "🌩"),
+    (227, "❄"),
+    (230, "❄"),
+    (248, " "),
+    (260, " "),
+    (263, "🌧"),
+    (266, "🌧"),
```
Comment (khaneliman) [inline src/constants.rs:17]: update to match 179

File: src/constants.rs [hunk 3]
```diff
@@ -1,64 +1,64 @@
...
+    (314, "🌧"),
+    (317, "🌧"),
+    (320, "🌨"),
+    (323, "🌨"),
+    (326, "🌨"),
+    (329, "🌨"),
+    (332, "🌨"),
+    (335, "🌨"),
+    (338, "🌨"),
+    (350, "🌨"),
```
Comment (khaneliman) [inline src/constants.rs:36]: update to match 182

File: src/constants.rs [hunk 4]
```diff
@@ -1,64 +1,64 @@
...
+    (353, "🌧"),
+    (356, "🌧"),
+    (359, "🌧"),
```
Comment (khaneliman) [inline src/constants.rs:39]: 179

File: src/constants.rs [hunk 5]
```diff
@@ -1,64 +1,64 @@
...
+    (362, "🌨"),
+    (365, "🌨"),
+    (368, "🌨"),
+    (371, "🌨"),
+    (374, "🌨"),
+    (377, "🌨"),
```
Comment (khaneliman) [inline src/constants.rs:45]: 182

File: src/constants.rs [hunk 6]
```diff
@@ -1,64 +1,64 @@
...
+    (404, "🌨"),
+    (407, "🌨"),
+    (410, "🌨"),
+    (413, "🌨"),
+    (416, "🌨"),
+    (419, "🌨"),
+    (422, "🌨"),
+    (425, "🌨"),
+    (428, "🌨"),
+    (431, "🌨"),
```
Comment (khaneliman) [inline src/constants.rs:61]: 182

File: src/constants.rs [hunk 7]
```diff
@@ -1,64 +1,64 @@
...
+    (113, "󰖙"),
+    (116, "󰖕 "),
+    (119, " "),
+    (122, " "),
+    (143, " "),
+    (176, " "),
+    (179, " "),
+    (182, " "),
+    (185, " "),
+    (200, "🌩"),
```
Comment (khaneliman) [inline src/constants.rs:11]: no icon for this?

**Review by khaneliman — CHANGES_REQUESTED** [review]
I like the concept of the change, but it feels incomplete to not offer an option to keep the previous style. Could basically create a config option and an alternate vector set and we choose which set to pull icons from based on the config.

File: src/constants.rs (hunk 7 above)
Comment (4rtemis-4rrow) [reply src/constants.rs:11]: there are icons there, wdym?

Comment (khaneliman) [reply src/constants.rs:11]: 󰖓

Comment (khaneliman) [reply src/constants.rs:11]: Doesn't look changed in PR 
image

Comment (4rtemis-4rrow) [reply src/constants.rs:11]: in either case, I just fixed the non-matching icons, and changes the icons in main.rs too, that way the UI is better unified

Comment (khaneliman) [reply src/constants.rs:11]: all the others look good, just these last 2 lightning icons

Comment (khaneliman) [reply src/constants.rs:11]: image

Comment (khaneliman) [reply src/constants.rs:11]: looks like some extra spaces in there too

Comment (4rtemis-4rrow) [reply src/constants.rs:11]: image

weird

Comment (khaneliman) [reply src/constants.rs:11]: How are you grabbing the icons? What font are you using locally and in browser? Is it using the latest nerd fonts ?

Comment (4rtemis-4rrow) [reply src/constants.rs:11]: jetbrains mono nerd, latest, as for browser, floorp's default (noto serf)

as for how I'm grabbing them, kitty's unicode input kitten,

**Comment by bjesus** [comment]
thanks for the PR. unfortunately, this will never be merged unless it's behind a flag. I personally don't like the idea of replacing a standard unicode character with a non-standard (and unstable, as happens something with nerdfont) hack, but I'm happy to support it if it was _optional_. note that the whole point of using unicode is that if you don't like the emojis design you're welcome to change font.

**Pull request closed without merging**